compiler/src/dotty/tools/dotc/ast/Positioned.scala

```diff
@@ -16,10 +16,12 @@ abstract class Positioned extends DotClass with Product {
 
   setPos(initialPos)
 
-  /** The item's position.
-   */
+  /** The item's position. */
   def pos: Position = curPos
 
+  /** The item's coord. */
+  def coord: Coord = curPos.toCoord
```

Comment:
I think this is wrong. "Coord" is an optimization for Symbols, so that they can take alternatively a Position or an Index. It's wrong to pollute other data types with this distinction - they should have no idea what a `Coord` is nor how to covert to it.